types.go

```diff
@@ -1361,6 +1361,23 @@ func (rr *CSYNC) len(off int, compression map[string]struct{}) int {
 	return l
 }
 
+// ZONEMD RR, from draft-ietf-dnsop-dns-zone-digest
+type ZONEMD struct {
+	Hdr    RR_Header
+	Serial uint32
+	Scheme uint8
```

Comment:
There should probably be constants for this and Hash.

Reply:
done in #edcbb0b